docs/admin/baremetal-installation-guide.md

```diff
@@ -240,3 +244,28 @@ If you run into an issue with migration `V0611`, ensure that the user running Fl
 
 If you encounter other errors migrating the database, make sure that you are using the recommended version of Flyway
 (see [Prerequisites](#prerequisites)).
+
+# Server Deployment Troubleshooting
+
+* Whenever unexpected behaviour arises, be sure to check the build logs, located at `${CATALINA_BASE}/logs`. Read through the output of `catalina.out`, `catalina.<date>.log`, and `localhost.<date>.log`.
+* Ensure you are using the correct version of Java with Tomcat. This can be checked in `catalina.out`. In case you are not, append/modify the value of `$JAVA_HOME` in `setenv.sh` or `/etc/default/tomcat9`. 
+* If file access for `/storage/miso` is causing an error, [this](https://stackoverflow.com/questions/56827735/how-to-allow-tomcat-war-app-to-write-in-folder) may help.
+
+
+# Development Alternatives
+
+ If you are don't have the correct version of MySQL installed, this alternative allows the same result without you downloading it. To do this, download [Docker](https://docs.docker.com/get-docker/) to use a Docker container. Creating the container:
+
+    docker run --name $CONTAINER_NAME -e MYSQL_ROOT_PASSWORD=$ROOT_PASSWORD -e MYSQL_DATABASE=$DB_NAME -e MYSQL_USER=$DB_USERNAME -e MYSQL_PASSWORD=$DB_PASSWORD -p 3306:3306 -d mysql:$MYSQL_VERSION
+
+
+Where:
+* `$CONTAINER_NAME` is your desired Docker container name.
+* `$ROOT_PASSWORD` is the root password to your MySQL.
+* `$DB_NAME` is the name of the database (e.g. "lims").
+* `$DB_USERNAME` is the username to access the database (e.g. "tgaclims").
+* `$DB_PASSWORD` is the password to access the database (e.g. "tgaclims").
+* `$MYSQL_VERSION` is the version of MySQL for the database (e.g. "5.7").
```

Comment:
Put the version in the command instead of as a variable - you must use the version that MISO is compatible with. You can also specify 8.0 since this will be in the same release as that upgrade